src/services/study/security-analysis.js

```diff
@@ -85,3 +100,24 @@ export function fetchDefaultSecurityAnalysisProvider() {
     console.debug(url);
     return backendFetchText(url);
 }
+
+export function getSecurityAnalysisParameters(studyUuid) {
+    console.info('get security analysis parameters');
+    const url = getStudyUrl(studyUuid) + '/security-analysis/parameters';
+    console.debug(url);
+    return backendFetchJson(url);
+}
+
+export function setSecurityAnalysisParameters(studyUuid, newParams) {
+    console.info('set security analysis parameters');
+    const url = getStudyUrl(studyUuid) + '/security-analysis/parameters';
+    console.debug(url);
+    return backendFetch(url, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(newParams),
```

Comment:
Isn't there a refactor with all other requests possible here ?  
things like header and how to parse the body can be simplified (with a utility method or a spread operator on a model object)

Reply:
I think that is better to refactor this with another Technical story, because with this big refacto for rest-api, I afraid to have problems. thank you

Reply:
okay